packages/f/fooyin/package.yml

```diff
@@ -0,0 +1,30 @@
+name       : fooyin
+version    : 0.4.5
+release    : 1
+source     :
+    - https://github.com/fooyin/fooyin/archive/refs/tags/v0.4.5.tar.gz : d151807ccbff641caa791df45ee042ae508e45ec6e095af15d1f201d459e4ed8
+homepage   : https://www.fooyin.org/
+license    : GPL-3.0-or-later
+component  : multimedia.audio
+summary    : fooyin is a music player built around customisation.
+description: |
+    fooyin is a music player built around customisation.
+networking : yes
```

Comment:
Why does it needs networking?

Reply:
> Why does it needs networking?

Networking can be disabled. Instead `kdsingleapplication-devel` needs to be added as a build dep. It couldn't find a system lib for it so it was pulling it as a git submodule.

```
-- Using 3rd-party KDSingleApplication
[1/9] Creating directories for 'kdsingleapplication-populate'
[1/9] Performing download step (git clone) for 'kdsingleapplication-populate'
Cloning into 'kdsingleapplication-src'...
fatal: unable to access 'https://www.github.com/KDAB/KDSingleApplication.git/': Could not resolve host: www.github.com

CMake Error at kdsingleapplication-subbuild/kdsingleapplication-populate-prefix/tmp/kdsingleapplication-populate-gitclone.cmake:39 (message):
  Failed to clone repository:
  'https://www.github.com/KDAB/KDSingleApplication.git'
```

Reply:
Then, it probably better to use git (and the commit hash) as the source instead of tarball

Reply:
Just need to add the missing dep and git isn't needed at all.